### PR TITLE
Add material for 2nd tutorial

### PR DESCRIPTION
**Pull request by danieljschmidt**
I'll add the solution to the tutorial questions to the book_model.json file after the tutorial has taken place

### CODE REVIEW

File: notebooks/welfare_huggett.jl [hunk 1]
```diff
@@ -642,13 +642,11 @@ df.π = stationary_distribution(Q_star, hh.m, π₀);
...
+	ax = Axis(fig[1,1], xlabel = "k")
 	df_groups   = groupby(df, :z)
-	lin1 = lines!(df_groups[1].k, df_groups[1].π, color = :red)
-	lin2 = lines!(df_groups[2].k, df_groups[2].π, color = :blue)
-	Legend(fig[1, 2],
-    [lin1, lin2],
-    ["π(k,y=y¹)", "π(k,y=y²)"])
+	lines!(ax, df_groups[1].k, df_groups[1].π, color = :red, label="π(k,y=y¹)")
+	lines!(ax, df_groups[2].k, df_groups[2].π, color = :blue, label="π(k,y=y²)")
+	Legend(fig[1, 2], ax)
```
Comment (greimel) [inline notebooks/welfare_huggett.jl:649]: This shows how you can simplify the creation of a legend.

File: notebooks/welfare_huggett.jl [hunk 2]
```diff
@@ -644,12 +644,23 @@ let
 	fig = Figure(; resolution)
 	ax = Axis(fig[1,1], xlabel = "k")
 	df_groups   = groupby(df, :z)
-	lines!(ax, df_groups[1].k, df_groups[1].π, color = :red, label="π(k,y=y¹)")
-	lines!(ax, df_groups[2].k, df_groups[2].π, color = :blue, label="π(k,y=y²)")
+	lines!(ax, df_groups[1].k, df_groups[1].π, label="π(k,y=y¹)")
+	lines!(ax, df_groups[2].k, df_groups[2].π, label="π(k,y=y²)")
```
Comment (greimel) [inline notebooks/welfare_huggett.jl:648]: If you don't care about the exact colors, you can simplify even more.

File: notebooks/welfare_huggett.jl [hunk 3]
```diff
@@ -644,12 +644,23 @@ let
...
+# ╔═╡ 7ca1d382-eba3-4b1d-a4f9-f1b7040a9df4
+let
+	# Alternative implementation of above figure.
+	figure = (; resolution = (600, 300))
+
+	@chain df begin
+		data(_) * mapping(:k, :π, color = :z => nonnumeric => "income") * visual(Lines)
+		draw(; figure)
+	end
+end
```
Comment (greimel) [inline notebooks/welfare_huggett.jl:662]: Here is the same plot using `AlgebraOfGraphics`, just to advertise this as an alternative way of thinking. Feel free to remove these lines of code.

File: notebooks/welfare_huggett.jl [hunk 4]
```diff
@@ -823,6 +834,22 @@ let
...
+	df_big = vcat(df, df_eq, source = :type => ["PE", "GE"])
+	@chain df_big begin
+		data(_) * mapping(
+			:k, :Δ,
+			linestyle=:type => "equilibrium",
+			color=:z => nonnumeric => "income"
+		) * visual(Lines)
+		draw(; figure)
+	end
+end
```
Comment (greimel) [inline notebooks/welfare_huggett.jl:851]: Same: Simpler code using `AlgebraOfGraphics`. Feel free to remove again.

**Review by greimel — APPROVED** [review]
This looks great, thanks @danieljschmidt. I've added some suggestions for improving style but feel encouraged to go with your own version.

File: notebooks/welfare_huggett.jl (hunk 4 above)
Comment (greimel) [reply notebooks/welfare_huggett.jl:851]: image

**Comment by danieljschmidt** [comment]
Thanks for the help with the AoG package, @greimel! I have changed the code for all figures following your suggestions

**Pull request merged**